package.json

```diff
@@ -61,5 +63,8 @@
       "CLI",
       "backtick"
     ]
+  },
+  "peerDependencies": {
```

Comment:
It is better to create separate PR for another changes

Reply:
Fixed/